src/mlpack/methods/ann/layer/virtual_batch_norm.hpp

```diff
@@ -0,0 +1,194 @@
+/**
+ * @file virtual_batch_norm.hpp
+ * @author Saksham Bansal
+ *
+ * Definition of the Virtual Batch Normalization layer class.
```

Comment:
Please change it to `VirtualBatchNorm` to stay consistent with `MiniBatchDiscrimination`.